apps/sms/style/sms.css

```diff
@@ -829,6 +829,12 @@ Do not remove.
   background: #fff;
 }
 
+/* Adjust the name line-height to prevent bottom of chars been cut */
+.contact-list .suggestion > .name {
+  line-height: 2.4rem;
+  margin-top: -0.2rem;
```

Comment:
Does it look better with this ```margin-top```, with or without this it looks differently than in master anyway? Don't object, just asking, so it's up to you :)